requirements.txt

```diff
@@ -42,7 +42,7 @@ citeproc-py==0.4.0
 boto3==1.4.7
 django-waffle==0.13.0
 pymongo==3.7.1
-progressbar==2.3
+tqdm==4.28.1
```

Comment:
@Johnetordoff - quick comment - there's also a `release.txt` file with the `progressbar==2.3` - I assume we need to make this change there as well?

Reply:
Ah, good catch @pattisdr . Since `release.txt` included `-r ../requirements.txt`, we can just remove progressbar from `release.txt`. cc @Johnetordoff